### PR TITLE
fw_att_control: fix airspeed subscription and validity check

### PR DESCRIPTION
**Pull request by tstastny**
- fixed airspeed subscription that was not polling new updates properly resulting in the timestamp constantly =0 (as well as all other members of the airspeed struct)
- replaced unsigned integer subtraction with doubles on airspeed last update check

Without these fixes, the airspeed scaling would not be active (assumes the trim value when the airspeed update was "invalid"). Also, this keeps the subscriptions all defined in a consistent manner.

### CODE REVIEW

File: src/modules/fw_att_control/fw_att_control_main.cpp [hunk 1]
```diff
@@ -938,12 +955,12 @@ FixedwingAttitudeControl::task_main()
 				float airspeed;
 
 				/* if airspeed is non-finite or not valid or if we are asked not to control it, we assume the normal average speed */
-				const bool airspeed_valid = PX4_ISFINITE(_sub_airspeed.get().indicated_airspeed_m_s)
-							    && ((_sub_airspeed.get().timestamp - hrt_absolute_time()) < 1e6);
+				const bool airspeed_valid = PX4_ISFINITE(_airspeed.indicated_airspeed_m_s)
+							    && (fabs((double)_airspeed.timestamp - (double)hrt_absolute_time()) < 1.0e6);
```
Comment (dagar) [inline src/modules/fw_att_control/fw_att_control_main.cpp]: Why use doubles here? I would change this to `hrt_elapsed_time(&_airspeed.timestamp)`.

**Comment by dagar** [comment]
This was intentionally ugly to encourage moving things towards the newer uORB c++ wrappers that are usually safer and easier to use. I'll debug this to see where it's broken.

**Comment by dagar** [comment]
The core problem here is that fw_att_control_main allocates the FixedwingAttitudeControl (where _sub_airspeed is constructed), then kicks off the task to actually run. The file descriptor within the Subscription is wrong.

I'll quickly convert fw_att_control to use ModuleBase, where it's easier to not make these errors.

**Comment by dagar** [comment]
@tstastny https://github.com/PX4/Firmware/pull/8899 fixeds your issue and gets rid of more unnecessary module boilerplate.

File: src/modules/fw_att_control/fw_att_control_main.cpp (hunk 1 above)
Comment (tstastny) [reply src/modules/fw_att_control/fw_att_control_main.cpp]: yes - good call. please see the comment in #8899

**Comment by tstastny** [comment]
Addressed in #8899

**Pull request closed without merging**